.github/workflows/build.yaml

```
@@ -98,7 +98,25 @@ jobs:
         with:
           name: pages
           path: dist/design
-# TODO: build design system
+
+  publish-documentation:
+    if: github.ref == 'refs/heads/main'
```

It probably makes sense if we derive the default branch directly from GitHub, so it's in-sync with all of our protection and deployment rules (see also https://stackoverflow.com/a/69995861):

```suggestion
    if: github.ref_name == github.event.repository.default_branch
```